README.md

```diff
@@ -41,12 +38,12 @@
 Run the following to install this library:
 
 ```bash
-$ composer require laminas/laminas-text
+composer require laminas/laminas-text
 ```
 
 ## Documentation
 
-Browse the documentation online at https://docs.laminas.dev/laminas-text/
+Browse the documentation online at <https://docs.laminas.dev/laminas-text/>
```

Comment:
This is not necessary as the rule has been disabled.

Reply:
See: https://github.com/laminas/laminas-continuous-integration-action/blob/f474cab0f489b50c3b31d6ae04f0a5fc189c796d/setup/markdownlint/markdownlint.json#L9

Reply:
@froschdesign done